modular_zubbers/modules/code/alt_job_titles.dm

```diff
@@ -0,0 +1,421 @@
+/**
+ * This is the file you should use to add alternate titles for each job, just
+ * follow the way they're done here, it's easy enough and shouldn't take any
+ * time at all to add more or add some for a job that doesn't have any.
+ */
+
+/datum/job/ai
+	alt_titles = list(
+		"AI",
+		"Station Intelligence",
+		"Automated Overseer",
+		"Language Model",
+		"Mind Scan",
+		"Remote Helper Agent",
+	)
+
+/datum/job/assistant
+	alt_titles = list(
+		"Assistant",
+		"Civilian",
+		"Tourist",
+		"Businessman",
+		"Businesswoman",
+		"Trader",
+		"Entertainer",
+		"Freelancer",
+		"Artist",
+		"Off-Duty Staff",
+		"Off-Duty Crew",
+		"Collector",
+		"Maintenance Dweller",
+		"Hobbyist",
+		"Intern",
+		"Visitor",
+		"Gamer",
```

Comment:
This feels a little too no-RP for me.

Reply:
Some people might be visiting and want to do hobbies like arts and crafts or read books.
For the intern role, this one should make sense.
Same as the first here, some individuals might be here visiting, as non-workers.
There is a gamer den in maintenance, and arcade machines all over the station.

Essentially, this helps to differentiate the activities the person is planning on performing for that shift.